codegen.yml

```diff
@@ -0,0 +1,12 @@
+overwrite: true
+schema: "src/gqlSchema.json"
+documents: "src/queries.ts"
+generates:
+  src/graphqlTypes.ts:
+    plugins:
+      - "typescript"
+      - "typescript-operations"
+config:
+  maybeValue: T
```

Comment:
Denne er egentlig `T | null` fordi vi får null for manglende verdier fra graphql og ikke undefined. Sånn som jeg har gjort det nå blir null som vi får fra graphql behandlet som undefined. Da slipper vi å håndtere både for null og undefined. Det blir ikke helt riktig, men tviler på at det har noen effekt i praksis, og det er også sånn vi gjorde det før